Makefile

```diff
@@ -20,6 +20,10 @@ OVS_VERSION        := $(shell head -n 1 build/images/deps/ovs-version)
 GO_VERSION         := $(shell head -n 1 build/images/deps/go-version)
 CNI_BINARIES_VERSION := $(shell head -n 1 build/images/deps/cni-binaries-version)
 BUILD_TAG          := $(shell build/images/build-tag.sh)
+NANOSERVER_VERSION := $(shell head -n 1 build/images/deps/nanoserver-version)
+WIN_BUILD_TAG      := $(shell echo $(GO_VERSION) $(CNI_BINARIES_VERSION) $(NANOSERVER_VERSION)|md5sum|head -c 10)
+WIN_OVS_VERSION	   := $(shell head -n 1 build/images/deps/ovs-version-windows)
+WIN_BUILD_OVS_TAG  := $(NANOSERVER_VERSION)-$(WIN_OVS_VERSION)
```

Comment:
you seem to be adding back lines which have been removed recently in #6325 by @wenyingd 
is this intentional? it seems unrelated to the rest of the PR

Reply:
in particular, file `build/images/deps/nanoserver-version` no longer exists

Reply:
thanks for pointing this out will remove these.